app/Models/Book.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use ApiPlatform\Metadata\ApiProperty;
+use ApiPlatform\Metadata\ApiResource;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Symfony\Component\Serializer\Attribute\Groups;
+
+#[ApiResource(
+    types: ['https://schema.org/Book'],
+    normalizationContext: ['groups' => 'book'],
+    denormalizationContext: ['groups' => 'book'],
+)]
+#[ApiProperty(property: 'title', serialize: new Groups(['book']))]
+#[ApiProperty(property: 'isbn', serialize: new Groups(['book']))]
+#[ApiProperty(property: 'isbn', serialize: new Groups(['book']))]
+#[ApiProperty(property: 'description', serialize: new Groups(['book']))]
```

Comment:
@dunglas This part gives me an error when I run `php artisan migrate` and a fatal error on the API doc,  as you can see below:
![image](https://github.com/user-attachments/assets/d332ec0e-f7f0-494d-88d4-1c11a21b9117)
![image](https://github.com/user-attachments/assets/d8bc89ac-0ecc-437c-88df-0ee8df59b27c)

Reply:
I fixed this on core. We just need to update the deps.

Reply:
Oh, perfect, so I'll do it in PR #4 👌

Reply:
Rebase ready @dunglas